mypy/semanal.py

```diff
@@ -1275,6 +1273,11 @@ def analyze_metaclass(self, defn: ClassDef) -> None:
             # do not declare explicit metaclass, but it's harder to catch at this stage
             if defn.metaclass is not None:
                 self.fail("Inconsistent metaclass structure for '%s'" % defn.name, defn)
+        else:
+            if defn.info.metaclass_type.type.fullname() == 'enum.EnumMeta':
```

Comment:
So if the metaclass is a subclass of EnumMeta, it's not an enum? (I'm not sure I care, but I wonder if this is an intentional choice.)